java-client/session/src/main/java/io/deephaven/client/impl/Export.java

```diff
@@ -31,7 +31,12 @@ public final class Export implements AutoCloseable, HasExportId {
 
     @Override
     public ExportId exportId() {
-        return new ExportId("Table", state.exportId());
+        return new ExportId(TableObject.TYPE, state.exportId());
```

Comment:
Maybe we should deprecate this class? It's name-squatting on something generic, but it's table-specific.

Reply:
https://github.com/deephaven/deephaven-core/issues/4534